Assets/MirageXR/Services/Moodle/MoodleManager.cs

```diff
@@ -264,7 +271,13 @@ private static List<Session> ParseArlemListJson(string json)
             }
             catch (Exception e)
             {
-                Debug.LogError($"[MoodleManager] ParseArlemListJson error\nmessage: {e}");
+                RootView_v2.Instance.dialog.ShowMiddle(
+                    "Network Error",
+                    "Could not connect to server, showing local activities only.",
+                    "OK", () => AppLog.LogInfo("[MoodleManager] User acknowledged network error"),
+                    "Cancel", () => AppLog.LogInfo("[MoodleManager] User acknowledged network error"),
+                    true);
```

Comment:
```suggestion
                RootView_v2.Instance.dialog.ShowMiddle(
                    "Network Error",
                    "Could not connect to server, showing local activities only.",
                    "OK", () => AppLog.LogInfo("[MoodleManager] User acknowledged network error")
                    true);
```

the dialog now supports a one-button window